gammapy/datasets/load.py

```diff
@@ -119,6 +120,18 @@ def load_psf_fits_table():
     return fits.open(filename)
 
 
+def load_aeff2D_fits_table():
+    """Load an example aeff2D FITS file..
+
+    Returns
+    -------
+    hdu_list : `~astropy.io.fits.HDUList`
+        aeff2D file contents.
+    """
+    filename = get_path('irfs/aeff2D.fits')
```

Comment:
20K, git says however
The following paths are ignored by one of your .gitignore files:
gammapy/datasets/data/irfs/aeff2D.fits